docs/detections/api/rules/rules-api-bulk-actions.asciidoc

```diff
@@ -688,3 +718,56 @@ Notice that there are no arrays in `attributes.results`. In dry run mode, rule u
     }
 }
 --------------------------------------------------
+
+[discrete]
+[[actions-object-schema-bulk]]
+===== `actions` schema
```

Comment:
Is this section going to be deleted?

Reply:
I think it shouldn't. See Joe's comment above, but I would have liked to extract the actions schema to its own page and link to it from both this page and from [this page](https://github.com/jpdjere/security-docs/blob/main/docs/detections/api/rules/rules-api-bulk-actions.asciidoc#L334), but he mentioned that he'd rather not do these refactoring before the docs are migrated, soon.

And I think the actions schema should be in this page, otherwise it's confusing to understand what it means in the context of this API request.

Reply:
I see. The reason I asked is that when I was reviewing this PR links to the actions schema were pointing to the rule creation page. Now when they send the user to this section it LGTM 👍

Reply:
What I'd probably do is I'd lift "`actions` schema" and "`throttle` schema" up, because right now the page structure is:

1. "BulkEditAction object" - its schema
2. Examples
3. "`actions` schema"
4. "`throttle` schema"

I'd keep examples after all schemas

Reply:
Makes sense, reorganized page